stories/Drawer.js

```diff
@@ -1,18 +1,21 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
-import { withKnobs, text } from '@storybook/addon-knobs';
+import { withKnobs, text, boolean } from '@storybook/addon-knobs';
 import { Drawer } from '@textkernel/oneui';
 
 storiesOf('Atoms|Drawer', module)
     .addDecorator(withKnobs)
-    .add('Drawer', () => (
-        <Drawer
-            initialIsOpen
-            title={text('Drawer title', '12/13 processed (2 need review, 1 failed)')}
-        >
-            {text(
-                'Content',
-                `
+    .add(
+        'Drawer',
+        () => (
+            <Drawer
+                isShown={boolean('isShown', true)}
+                isExpanded={boolean('isExpanded', true)}
```

Comment:
can we demonstrate how the component behaves with `onClick` prop defined? We can add that prop conditionally: 
```
onClick={
  boolean('Use onClick prop', false)
    ? () => console.log('onClick callback has been called')
    : undefined
}
```